coordinator/src/resource_manager.rs

```diff
@@ -286,7 +290,6 @@ impl ResourceManager {
         let names = resourceinfo.get_names();
         if names.len() == 4 {
```

Comment:
Try to avoid this kind of logic without clear semantics

Reply:
optimizations will be made later